doc/configuration.rst

```diff
@@ -382,6 +382,29 @@ release
   "cdh4".
 
 
+[k8s]
```

Comment:
> Both k8s and Kubernetes are mutually intelligible to Kubernetes users, but I guess that would be better for continuity.

I want luigi users who don't know kubrenetes to find the docs to not have any surprises as well. :)